docs/HACKING.md

```diff
@@ -105,8 +107,7 @@ See [the build instructions](#build) for more information on building the image.
 
 You can then run the following command in the root of this git repository:
 ```shell script
-operator-sdk bundle create $BUNDLE_REPOSITORY:$BUNDLE_TAG -d deploy/olm-catalog/windows-machine-config-operator/manifests \
---channels alpha --default-channel alpha --image-builder podman
+make bundle-build BUNDLE_IMG=<BUNDLE_REPOSITORY:$BUNDLE_TAG>
```

Comment:
Is there no way to specify the channel now?

Reply:
That is done within the bundle metadata